spec.html

```diff
@@ -588,7 +588,7 @@ <h1>Terminal Symbols</h1>
         <p>In contrast, in the syntactic grammar, a contiguous run of fixed-width code points is a single terminal symbol.</p>
         <p>Terminal symbols come in two other forms:</p>
         <ul>
-          <li>In the lexical and RegExp grammars, Unicode code points without a conventional printed representation are instead shown in the form "&lt;ABBREV>" where "ABBREV" is a mnemonic for the code point or set of code points. These forms are defined in <emu-xref href="#sec-unicode-format-control-characters" title></emu-xref>, <emu-xref href="#sec-white-space" title></emu-xref>, and <emu-xref href="#sec-line-terminators" title></emu-xref>.</li>
+          <li>In the lexical and RegExp grammars, Unicode code points without a conventional printed representation are instead shown in the form "&lt;U+0000 (NULL)>" where `0000` is 4 to 6 hexits representing the code point in hexadecimal notation and `NULL` is the code point name.</li>
```

Comment:
This seems gratuitously divergent from [Unicode conventions](https://www.unicode.org/versions/Unicode15.1.0/appA.pdf#G7496). Should we instead try to align?

Reply:
Are you saying we should use small caps? As for the name, I chose to use one of the official aliases when I felt it was more appropriate/descriptive. I can explicitly state that it is the code point name or an alias if you prefer.

Reply:
Yes, I think we should use small caps and avoid brackets except for sequences, e.g.
```suggestion
          <li>In the lexical and RegExp grammars, Unicode code points without a conventional printed representation are instead shown in the form "U+0000 <small class="code-point-name">NULL</small>" where `0000` is 4 to 6 hexadecimal digits representing the code point and `NULL` is the code point name.</li>
```
or maybe ecmarkup support
```suggestion
          <li>In the lexical and RegExp grammars, Unicode code points without a conventional printed representation are instead shown in the form "<code data-code-point="U+0000">NULL</code>" where `0000` is 4 to 6 hexadecimal digits representing the code point and `NULL` is the code point name.</li>
```
or even ecmarkdown
```suggestion
          <li>In the lexical and RegExp grammars, Unicode code points without a conventional printed representation are instead shown in the form "U+0000 ^^NULL^^" where `0000` is 4 to 6 hexadecimal digits representing the code point and `NULL` is the code point name.</li>
```

Reply:
I think it's fine to defer the small-caps names (with possible tooling support) to a follow-up.

Reply:
Yeah, I think so. :+1: